framework/console/console.cc

```diff
@@ -96,7 +96,7 @@ Console::LuaWrapperCall(lua_State* L)
       main_arguments_block.AddParameter(arg_name, lua_tostring(L, p));
     else if (lua_istable(L, p))
     {
-      auto block = chi_lua::TableParserAsParameterBlock::ParseTable(L, p);
+      auto block = TableParserAsParameterBlock::ParseTable(L, p);
```

Comment:
@wdhawkins  @zhardy-lanl @andrsd 
Previously, someone reading this would know that TableParserAsParameterBlock was related to lua. By flattening the namespace, that connection is lost. I am fairly fine with this but I have two comments:
1. if we want to increase the clarity/hints, should such a function to named LuaTableParserAsParameterBlock  ?
2. a more general question: how will another input system (Moose system, or python API) would interfere or co-exist here ? (likely a more general discussion for another meeting ...)

Reply:
The console object will be moved into the lua layer in one of the upcoming PRs I have prepared. It will end up sitting in the `opensnlua` namespace, so we won't need to rename this object or anything else related to lua. 

The goal of issue #9 is to have all lua-related stuff in the lua layer and have the `framework` and `modules` agnostic of the outer layer using it. So things like a python wrapper and whatnot will sit in their own location using the C++ API.

Reply:
In case you want to check how the code is going to be structured to accomplish the lua/core separation, you can look at my [next branch](https://github.com/andrsd/opensn/tree/next) in my fork...

Reply:
What does everyone think of `opensn::lua` instead? It seems cleaner than `opensn<api>`. I know we agreed on a single layer, but for APIs, I think a single nesting makes sense here.